macrosynergy/management/update_df.py

```diff
@@ -13,43 +14,94 @@ def update_df(df: pd.DataFrame, df_add):
     :param <pd.DataFrame> df_add: dataframe with the latest values. The categories will
         either be appended to the aggregate dataframe, or replace the previously defined
```

Comment:
:param <pd.DataFrame> df_add: dataframe that will be appended to existing standardized dataframe and replace any existing tickers, combinations of cross sections and categories with the same names.